Pegasus/include/pegasus/Scene.hpp

```diff
@@ -32,7 +32,7 @@ class Scene
      * @brief Runs physical simulation with the given duration
      * @param duration delta time of the integration in seconds
```

Comment:
```suggestion
     * @param duration integration delta time
```
Are you locked to seconds or is it an arbitrary metric?

Reply:
I am looked to seconds in this interface.